### PR TITLE
Allison/update-prompts-daily

### PR DESCRIPTION
**Pull request by alz96lu**
### **Changes:**

1. Created new collection in firebase DB called "days" where the prompts are stored
_- alphabetical order a little annoying..._

2. Created a method for updating prompts for the database in dataService.js.
_- consists of deleting the current one and fetching the new one_

### CODE REVIEW

**Comment by jjang4001** [comment]
I don't think we should be updating the db from the client side, but we can talk about how to do it instead tonight?

**Comment by jjang4001** [comment]
Disregard deleted comment lol

File: src/services/data-service.js [hunk 1]
```diff
@@ -1,11 +1,41 @@
...
+
+  //sets the prompt's state to the same one as DB
+  displayPromptsFromDB(prompt, state) {
+    state.setState({
+      title: prompt
+    });
+  }
+
+  updatePromptsFromDB(state, state_title) {
+    firebaseReference.collection('days').doc(state_title).delete(); //deletes the old state
```
Comment (jjang4001) [inline src/services/data-service.js]: Does this delete the previous prompt from the db?

Comment (alz96lu) [reply src/services/data-service.js]: yes!

Comment (jjang4001) [reply src/services/data-service.js]: Do we want it to if we have an archives section?

Comment (jjang4001) [reply src/services/data-service.js]: I also don't think client-side code should be responsible for deleting prompts.

Comment (alz96lu) [reply src/services/data-service.js]: hmmm interesting, discuss at meeting?

**Review by parkhk10 — APPROVED** [review]

**Pull request merged**